docs/UserGuide.md

```diff
@@ -491,15 +497,15 @@ If you entered the wrong class name or would like to change the name of a class,
 * Edits the class at the specified `INDEX`.
 
 For example:
-* Your module coordinator decides to allocate another class to you and you want to be able to distinguish between
- the two classes. Hence you decide to edit the `CS2103T Tutorial` class to `CS2103T Tutorial T10`. You type the
- command `edit-class 1 n\CS2103T Tutorial T10` and press <kbd>Enter</kbd>.
+1. Your module coordinator decides to allocate another class to you and you want to be able to distinguish between
+   the two classes. Hence you decide to edit the `CS2103T Tutorial` class to `CS2103T Tutorial T10`. You type the
+   command `edit-class 1 n\CS2103T Tutorial T10` and press <kbd>Enter</kbd>.
```

Comment:
This step is a little long, perhaps we can put the `You type...` into a separate step on its own? Maybe something like this?

```suggestion
1. Your module coordinator decides to allocate another class to you and you want to be able to distinguish between
   the two classes. Hence you decide to edit the `CS2103T Tutorial` class to `CS2103T Tutorial T10`. 

1. Type the command `edit-class 1 n\CS2103T Tutorial T10` and press <kbd>Enter</kbd>.
```